README.md

```diff
@@ -1,33 +1,33 @@
 # Arrayzy
 
-A native PHP arrays easy manipulation library in OOP way.
+A native PHP library for manipulating arrays in an OOP way.
 
 [![SensioLabsInsight](https://insight.sensiolabs.com/projects/e0235f5d-a89b-4add-b3c6-45813d2bf9eb/mini.png)](https://insight.sensiolabs.com/projects/e0235f5d-a89b-4add-b3c6-45813d2bf9eb)
 [![Build Status](https://travis-ci.org/bocharsky-bw/Arrayzy.svg?branch=master)](https://travis-ci.org/bocharsky-bw/Arrayzy)
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/bocharsky-bw/Arrayzy/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/bocharsky-bw/Arrayzy/?branch=master)
 [![Code Coverage](https://scrutinizer-ci.com/g/bocharsky-bw/Arrayzy/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/bocharsky-bw/Arrayzy/?branch=master)
 
-There are two available classes with a different behaviour:
+There are two classes available with different behavior:
 
 * [Arrayzy\MutableArray](#mutablearray)
 * [Arrayzy\ImmutableArray](#immutablearray)
 
 ## MutableArray
 
-Each functional method operate on the same array and return the same instance
-(**DO NOT** create a new instance) except only few methods starts with `create` prefix.
-This way a bit improve performance and provide more convenience usage in OOP way.
+Each method operates on the same array and returns the array itself
+(it **DOES NOT** create a new instance), except methods starting with the
+`create` prefix. This way has slightly better performance and is more
+convenient to use in an OOP way.
 
-> **NOTE:** Check the [CreateClone](#createclone) section if you want to operate on new instance NOT on same instance.
+> **NOTE:** Check the [CreateClone](#createclone) section if you want to operate on a new instance INSTEAD of the instance itself.
```

Comment:
How about just `Check the [CreateClone](#createclone) section if you want to operate on a new instance.`? Or perhaps `Check the [CreateClone](#createclone) section if you want to operate on a new instance to preserve the current one.`?